planner/core/testdata/analyze_suite_out.json

```diff
@@ -352,8 +352,8 @@
       "IndexLookUp(Index(t1.a)[[1,1]], Table(t1)->Sel([le(test.t1.b, 2)]))",
       "TableReader(Table(t2)->Sel([le(test.t2.a, 2)]))",
       "Analyze{Index(a),Index(b)}",
-      "TableReader(Table(t4)->Sel([le(test.t4.a, 2)]))",
-      "IndexReader(Index(t4.b)[[-inf,2)])",
+      "PartitionUnionAll{TableReader(Table(t4)->Sel([le(test.t4.a, 2)]))->TableReader(Table(t4)->Sel([le(test.t4.a, 2)]))}",
+      "PartitionUnionAll{IndexReader(Index(t4.b)[[-inf,2)])->IndexReader(Index(t4.b)[[-inf,2)])}",
```

Comment:
The reason for this change is that global-level stats is used to generate the execution plan. Now we only have partition-level stats.